pymeasure/instruments/hp/hp856Xx.py

```diff
@@ -498,6 +609,102 @@ def __init__(self, adapter, name="Hewlett-Packard HP856Xx", **kwargs):
+        the screen, execute the :meth:`preset` command."""
+        self.write("ADJCRT")
+
+    adjust_if = Instrument.control(
+        "ADJIF?", "ADJIF %s",
+        """
+        Control the automatic IF adjustment. This function is normally
+        on. Because the IF is continuously adjusting, executing the IF alignment routine is seldom
+        necessary. When the IF adjustment is not active, an "A" appears on the left side of the
+        display.
+
+        - FULL IF adjustment is done for all IF settings.
+        - CURR IF adjustment is done only for the IF settings currently displayed.
+        - OFF turns the continuous IF adjustment off.
+        - ON reactivates the continuous IF adjustment.
+
+        Type: :code:`bool, str`
+        """,
+        validator=strict_discrete_set,
+        map_values=True,
+        values={True: "1", False: "0", "FULL": "FULL", "CURR": "CURR"},
+        cast=str
+    )
+
+    trace_a_minus_b = Instrument.control(
+        "AMB?", "AMB %s",
+        """
+        Subtract the contents of trace B from trace A and places the result, in
+        dBm (when in log mode), in trace A. When in linear mode, the result is in volts. If trace
+        A is in clear-write or max-hold mode, this function is continuous. When AMB is active, an
+        "M" appears on the left side of the display. :attr:`trace_a_minus_b_plus_dl`  overrides AMB.
+
+        Type: :code:`bool`
+
+        .. warning::
+
+            The displayed amplitude of each trace element falls in one of 600 data points.
+            There are 10 points of overrange, which corresponds to one-sixth of a division
+            Kg of overrange. When adding or subtracting trace data, any results exceeding
+            this limit are clipped at the limit.
+        """,
+        validator=strict_discrete_set,
+        map_values=True,
+        values={True: "1", False: "0"},
+        cast=str
+    )
+
+    trace_a_minus_b_plus_dl = Instrument.control(
+        "AMBPL?", "AMBPL %s",
+        """
+        Subtract the contents of trace B from trace A, adds the display line
+        to this value, and stores the result in dBm (when in log mode) in trace A. When in linear
+        mode, the result is in volts. If trace A is in clear-write or max-hold mode, this function
+        is continuous. When this function is active, an "M" appears on the left side of the display.
+
+        Type: :code:`bool`
+
+        .. warning::
+
+            The displayed amplitude of each trace element falls in one of 600 data points.
+            There are 10 points of overrange, which corresponds to one-sixth of a division
+            Kg of overrange. When adding or subtracting trace data, any results exceeding
+            this limit are clipped at the limit.
+        """,
+        validator=strict_discrete_set,
+        map_values=True,
+        values={True: "1", False: "0"},
+        cast=str
+    )
+
+    annotation = Instrument.control(
```

Comment:
For boolean properties, it is good to form the property name as a question in order to facilitate to know, what happens:
```suggestion
    annotation_enabled = Instrument.control(
```

now `annotation_enabled = True` is self-evident.